lib/models/candidate.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from requests import HTTPError
+
+from lib.api.checkr_api import CheckrAPI
+from lib.helpers.checkr_api_error import CheckerAPIFailedCreation
+
+
+@dataclass
+class Candidate:
+    checkr_api: CheckrAPI
+    first_name: str
+    last_name: str
+    email: str
+    driver_license_number: str
+    driver_license_state: str
+    work_locations: list[dict[str, str]]
+
+    id: Optional[str] = None
+    middle_name: Optional[str] = None
+    no_middle_name: Optional[bool] = None
+    mother_maiden_name: Optional[str] = None
+    ssn: Optional[str] = None
+    dob: Optional[str] = None
+    object: Optional[str] = None
+    zipcode: Optional[str] = None
+    uri: Optional[str] = None
+    phone: Optional[str] = None
+    geo_ids: Optional[list[str]] = None
+    previous_driver_license_number: Optional[str] = None
+    previous_driver_license_state: Optional[str] = None
+    copy_requested: Optional[bool] = None
+    custom_id: Optional[str] = None
+    report_ids: Optional[list[str]] = None
+    adjudication: Optional[str] = None
+
+    def create(self, raise_on_failure: bool = True) -> bool:
+
+        try:
+            response = self.checkr_api.create_candidate(
```

Comment:
It's poor practice to have long running `try` scopes. What I suggest is this:

```python
def create(self, raise_on_failure: bool = True) -> bool:
    try:
        self._create_via_api()
    except HTTPError:
        if raise_on_failure:
            raise
        return False

def _create_via_api(self):
    self.checkr_api.create_candidate( ... )
```

eThe underscore here means that it's not recommended to call directly.

Reply:
I see. Thanks for the suggestion!

Reply:
It also solves the issue with the max lines. Amazing!